requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 pyuca==1.1
+wrapt==1.10.5
```

Comment:
@feanil -- 
About library- the `wrapt` is used to avoid conflicts between `@contract` and any parameterized decorator. `setuptools.wraps()` fails to do so, but `@wrapt.decorator` does it perfectly. 
IMO, `wrapt`  is better than `setuptools.wraps()`. 

Is it going to be used on production?
- Yes. The library Call Stack Manager is used to get unique, unknown call stacks of `StudentModule` and `StudentModuleHistory` on production. Due to this, we would be able to  make sure that all call to `studentmodule` and `studentmodulehistory` will go through `edx-user-state-client` interface.